.gitattributes

```diff
@@ -1,10 +1,15 @@
-* text eol=lf
-*.jpg -text
-*.png -text
-*.gif -text
```

Comment:
I mean we should leave them because we get several problems with media files inside the repository if we remove them.